ddtrace/settings/config.py

```diff
@@ -286,6 +286,8 @@ def __init__(self):
                 log.warning("Invalid obfuscation pattern, disabling query string tracing")
                 self.http_tag_query_string = False  # Disable query string tagging if malformed obfuscation pattern
 
+        self.ci_visibility_agentless_enabledd = asbool(os.getenv("DD_CIVISIBILITY_AGENTLESS_ENABLED", default=False))
```

Comment:
If we prefix this configuration with an `_` we will make it "internal". This could give us flexibility if we decide to rename or move this configuration. 

```suggestion
        self._ci_visibility_agentless_enabledd = asbool(os.getenv("DD_CIVISIBILITY_AGENTLESS_ENABLED", default=False))
```